lib/quandl/model/base.rb

```diff
@@ -0,0 +1,31 @@
+module Quandl
+  class ModelBase
+    def initialize(data, _options = {})
+      @raw_data = Hash[data.map { |k, v| [Quandl::Util.methodize(k), v] }]
+    end
+
+    def data_fields
+      @raw_data.keys.map(&:to_s)
+    end
+
+    def column_names
+      @raw_data.keys.map(&:to_s).map(&:titleize)
+    end
+
+    def to_a
+      @raw_data.values
+    end
+
+    def inspect
+      @raw_data.to_s
+    end
+
+    private
+
+    def method_missing(method_name, *args, &block)
+      return @raw_data[method_name.to_s] if @raw_data.key?(method_name.to_s)
```

Comment:
consider using indifferent access and removing this line? if it doesn't whatever